src/dodal/devices/dcm.py

```diff
@@ -0,0 +1,88 @@
+from bluesky.protocols import HasHints, Hints
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.motion import Motor
+from ophyd_async.epics.signal import epics_signal_r
+
+from dodal.beamlines.beamline_parameters import get_beamline_parameters
+
+
+class DCM(StandardReadable, HasHints):
```

Comment:
Should: I appreciate this wasn't your doing but while where here... Can we roll `-MO-DCM-01:` into the prefix? Either by specifying it in the beamline file like we do in other devices or by just doing a `prefix = prefix + "-MO-DCM-01"` as the first thing in the init?